packages/kbn-performance-testing-dataset-extractor/src/extractor.ts

```diff
@@ -39,6 +42,22 @@ export interface ScalabilitySetup {
   maxDuration: string;
 }
 
+const parsePayload = (hit: Document, log: ToolingLog): Promise<string | undefined> => {
+  let body;
+  if (
+    hit.http.request.body &&
+    hit.http.request.body.original &&
+    typeof hit.http.request.body.original === 'string'
```

Comment:
is there a possibility this can't be string ? I was thinking `hit.http.request.body?.original` would be enough

Reply:
I agree, will change